config/game_options.txt

```diff
@@ -517,7 +516,7 @@ ROUNDSTART_RACES plasmaman
 ##-------------------------------------------------------------------------------------------
 ## Uncommenting races will restrict them behind the patreon paywall
 
-#PAYWALL_RACES felinid
+#PAYWALL_RACES none
```

Comment:
I think this isn't needed, just a total deletion of this line should work

Reply:
True, will be added back with Simian species in future PR anyways